packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -514,7 +527,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         ISummarizerInternalsProvider
 {
     public get IContainerRuntime() { return this; }
-    public get IContainerRuntimeDirtyable() { return this; }
+    /**
+     * @deprecated 0.38 The IContainerRuntimeDirtyable interface and isMessageDirtyable() method will be removed in
+     * an upcoming release.
+     */
+    public get IContainerRuntimeDirtyable() {
+        console.error(`The IContainerRuntimeDirtyable interface and isMessageDirtyable() method are deprecated`);
```

Comment:
i would use the logger here and send to telemetry. logging on the end users device won't get the developers attention. i'm usually pretty aggressive here an log error

Reply:
Changed in latest